src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberFieldSymbol.cs

```diff
@@ -323,7 +323,7 @@ public sealed override bool HasInitializer
             get { return _hasInitializer; }
         }
 
-        public VariableDeclaratorSyntax VariableDeclaratorNode
+        private VariableDeclaratorSyntax VariableDeclaratorNode
```

Comment:
Generally, as a matter of software engineering hygiene, it is better to keep API surface area between components as small as possible. This is an example of the application of that principle.

---
In reply to: [80815616](https://github.com/dotnet/roslyn/pull/13999#discussion_r80815616) [](ancestors = 80815616)